src/resume.yaml

```
@@ -29,7 +29,7 @@ experience:
       - Webpack
       - NGINX
     summary:
-      - "A single-page web application to help my family & I to pick something to cook"
+      - "A single-page web application to help my family & I to pick something cook"
```

Removing the 'to' here seems to make the sentence make less sense to me here. Is that the suggestion you meant to make, or is it just a typo?

---

oh, that was the wrong `to`. I meant this:

```suggestion
      - "A single-page web application to help my family & I pick something to cook"
```